### PR TITLE
UT-Alignment-12d-6

### PR DESCRIPTION
**Pull request by LeeGregory12d**
## Unit test

Alignment with left arc in horizontal and crest and sag arcs for vertical.

## Covered topics

*tick one/many*

- [x] project setup
- [ ] georeferencing
- [x] alignment semantics
- [ ] alignment geometry
- [ ] linear placement
- [ ] project breakdown structure / spatial structure
- [ ] terrain
- [ ] triangulated irregular network
- [ ] open cross profile
- [ ] sectioned surface & solid
- [ ] pavement & course
- [ ] surface feature
- [ ] earthworks
- [ ] geotechnics
- [ ] longitudinal products (barriers, guardrail, ...)
- [ ] interchange
- [ ] structural
- [ ] drainage
- [ ] signage
- [ ] road furniture
- [ ] rail furniture
- [ ] ports & waterways furniture
- [ ] bridge furniture

### CODE REVIEW

File: UT-Alignment-12d-6/readme.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+
+### Intent
+
+IFC 4x3 RC2
+This scenario has an alignment with left arc in horizontal and crest and sag arcs for vertical
```
Comment (pjanck) [inline UT-Alignment-12d-6/readme.md]:
```suggestion
This scenario has an alignment with left arc in horizontal and crest and sag arcs for vertical.
```

**Review by pjanck — CHANGES_REQUESTED** [review]
Sentences usually end with punctuation. :)

**Comment by pjanck** [comment]
Following the changes to the schema, (we are now at `RC3`), we ask you to update your IFC files accordingly. The schema can be found [here](https://github.com/bSI-InfraRoom/IFC-Specification/blob/main/IFC4x3_RC3.exp). The checking procedure for IFC files has been updated - please merge in the current state of the `main` branch.

File: UT-Alignment-12d-6/readme.md (hunk 1 above)
Comment (LeeGregory12d) [reply UT-Alignment-12d-6/readme.md]: This has been fixed in RC3 version

**Comment by LeeGregory12d** [comment]
Stefan,
Was there anything else I had to change in the readme.md?

**Review by pjanck — APPROVED** [review]
Seen in the @jonm GeometryGym, seems fine.

**Pull request merged**